Reference/Templating/Mvc/examples.md

```diff
@@ -1,47 +1,28 @@
 ---
-versionFrom: 7.0.0
-needsV8Update: "true"
+versionFrom: 8.0.0
 ---
 
 # View/Razor Examples
 
 _Lots of examples of using various techniques to render data in a view_
 
-## Rendering a field with UmbracoHelper
-
-```csharp
-@Umbraco.Field("bodyContent")
-```
-
-## Rendering a field with UmbracoHelper with optional parameters
-
-```csharp
-@Umbraco.Field("bodyContent", insertBefore : "<h2>", insertAfter : "</h2>")
-```
-
 ## Rendering the raw value of a field from IPublishedContent
 
 ```csharp
-@Model.Content.Properties["bodyContent"].Value
+@Model.Properties["bodyContent"].Value
```

Comment:
I've tested this, and can't seem to get it working 😅
Might very well be doing something wrong .. In which context did you get an output using this?

Reply:
Facepalm! Let me fix them!

Reply:
This should now be fixed. :-)